packages/components/src/components/progress-indicator/_progress-indicator.scss

```diff
@@ -142,10 +142,13 @@
   }
 
   .#{$prefix}--progress-step--current svg {
-    width: 14px;
-    height: 14px;
+    stroke: $interactive-01;
     fill: $interactive-01;
     margin-top: rem(9.5px);
+
+    path:last-of-type {
```

Comment:
Just curious, what is this selector being used for? 👀

Reply:
Looks like it is being used to fill in the circle by increasing the width of the checked circle

Reply:
yeah...I couldn't find a Filled Circle Icon...so I'm improvised 😅

Reply:
> should I try to get a filled circle icon in the icon package instead?

👍

Reply:
cc @laurenmrice do you know of an icon we could use or will this work as is? 👀

Reply:
Yeah we do not have a filled circle icon. I think we can do this for now and I can talk with conrad about possibly adding one to our library for the future.